### DISCUSSION TITLE
The school Blocks the direct link to this Repository

### DISCUSSION THREAD

**Discussion by FearedFusionX** [3kh0/ext-remover#617] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I mean, I can bypass GoGuardian no problem because I can search up the repository on Github, but I probably would suggest renaming the Repository or creating a new one to prevent the direct link from being blocked.

**Comment by piplup7575** [comment]
simply get better?

**Reply by FearedFusionX** [reply]
thats what she said

**Comment by Hilololsusy** [comment]
Just copy and paste it or use a proxy or do one of the methods.

**Reply by FearedFusionX** [reply]
Well, I can go to the repository but the direct website is blocked.

**Comment by obkduishinijf** [comment]
better idea: proxy.

**Reply by FearedFusionX** [reply]
Yeah, I probably could but my school restricts me from changing the school network prefs
